scripts/prepare_sector_network.py

```diff
@@ -1878,15 +1893,61 @@ def add_services(n, costs):
         carrier="services biomass",
         p_set=p_set_biomass,
     )
-    co2 = (

```

Comment:
why is this kicked out?

Reply:
upon further research, I think we should not count the emissions from biomass as long as we set a "sustainable" biomass potential values. It is assumed that biomass emissions will be reabsorbed by forests/trees planted to replace those cut down to burn.

Reply:
This is also the same approach done in PyPSA-Eur

Reply:
We can set it as a config parameter and set its default value to false

Reply:
> upon further research, I think we should not count the emissions from biomass as long as we set a "sustainable" biomass potential values. It is assumed that biomass emissions will be reabsorbed by forests/trees planted to replace those cut down to burn.

I totally agree here.